gladier/tools/globus/compute.py

```diff
@@ -9,10 +9,30 @@
 ComputeFunctionType = t.Union[t.Callable[[t.Any], t.Any], str]
 
 
-class GlobusComputeState(ActionState):
+class ComputeState(ActionState):
+    """
+    The Globus Compute State allows for executing registered functions on a POSIX
```

Comment:
why the restriction to posix? Shouldn't this run any where Globus Compute runs? Is Globus Compute limited to posix?

Reply:
sounds good. I didn't realize Globus Compute only supported windows...